vagrant/provisioning/arkcase-ce-facts.yml

```diff
@@ -666,3 +666,34 @@ user_attribute_name: sAMAccountName
 
 microservice_protocol: http
 
+# foia_extension_formatted is only used by the foia play
+foia_extension_formatted: "{{ arkcase_extension_install | ternary('-' ~ arkcase_extension_id, '') }}"
+
+# arkcase_retrieve_strategy: "maven" if using nexus as artifacts provider 
+arkcase_artifact_repository_type: "release"
```

Comment:
Should we update the comment at line 181?

Reply:
Yes, if we want maven downloads. Again, nice catch!